apps/passport-client/src/dispatch.ts

```diff
@@ -942,3 +948,36 @@ async function promptToAgreePrivacyNotice(state: AppState, update: ZuUpdate) {
     });
   }
 }
+
+/**
+ * Formats the user's data as JSON, then creates a data URL link and
+ * simulates a click on that link in order to trigger a download.
+ */
+async function exportData(state: AppState, _update: ZuUpdate) {
+  const { self: user, pcds, subscriptions } = state;
+
+  // Since we already use this data for remote sync, we know that it's
+  // sufficient for loading an account on to a new device.
+  const { serializedStorage, storageHash } = await serializeStorage(
+    user,
+    pcds,
+    subscriptions
+  );
+
+  // Data in a data URL must be Base64-encoded
```

Comment:
Does this mean the resulting file the user saves is Base64 encoded, or is the encoding just a temporary artifact, stripped later?  I think it would be a nice feature if the result of export was human-readable, so people can see what PCDs are (similar to what's copied to the clipboard from a Frog PCD).

Reply:
> is the encoding just a temporary artifact, stripped later

pretty sure it's this

Reply:
Yes, this is part of how [data URLs](https://developer.mozilla.org/en-US/docs/web/http/basics_of_http/data_urls) work.